connectors/egeria/src/main/resources/json/asset/asset_metadata.json

```diff
@@ -1,82 +1,141 @@
-    },
-    "relatedAssetCount" : 1,
-    "ratingsCount" : 0
- }
+    "certificationCount": 0,
+    "class": "AssetResponse",
+    "commentCount": 0,
+    "connectionCount": 1,
+    "externalIdentifierCount": 0,
+    "externalReferencesCount": 0,
+    "informalTagCount": 1,
+    "keywordCount": 0,
+    "knownLocationsCount": 0,
+    "licenseCount": 0,
+    "likeCount": 0,
+    "noteLogsCount": 0,
+    "ratingsCount": 0,
+    "relatedAssetCount": 0,
+    "relatedHTTPCode": 200,
+    "relatedMediaReferenceCount": 0,
+    "schemaType": {
+        "attributeCount": 11,
+        "calculatedValue": false,
+        "class": "ComplexSchemaType",
+        "classifications": [
+            {
+                "class": "ElementClassification",
+                "classificationName": "Anchors",
+                "classificationOrigin": "ASSIGNED",
+                "classificationProperties": {
+                    "anchorGUID": "557b699f-0ce0-484d-bf1e-fec2c8f198c8"
+                },
+                "classificationTypeVersion": 0,
+                "elementCreatedBy": "garygeeke",
+                "elementCreateTime": "2021-02-19T08:04:55.959+00:00",
+                "elementMetadataCollectionId": "08c4bcd4-7485-4364-8275-1afce66e7653",
+                "elementMetadataCollectionName": "mds1",
+                "elementOrigin": "LOCAL_COHORT",
+                "elementSourceServer": "mds1",
+                "elementVersion": 1,
+                "headerVersion": 0,
+                "status": "ACTIVE"
+            }
+        ],
+        "extendedProperties": {
+            "isDeprecated": false,
+            "qualifiedName": "SchemaOf:{\"data_store\":\"S3\", \"bucket\":\"mybucket\", \"endpoint\":\"s3.eu-de.cloud-object-storage.appdomain.cloud\", \"object_key\":\"transactions.csv\", \"region\":\"eu\", \"data_location\": \"US\"}"
+        },
+        "guid": "7c2909e4-be17-41aa-9bb5-d25156833b83",
+        "headerVersion": 0,
+        "isDeprecated": false,
+        "type": {
+            "class": "ElementType",
+            "elementCreatedBy": "garygeeke",
+            "elementCreateTime": "2021-02-19T08:04:55.970+00:00",
+            "elementMetadataCollectionId": "08c4bcd4-7485-4364-8275-1afce66e7653",
+            "elementMetadataCollectionName": "mds1",
+            "elementOrigin": "LOCAL_COHORT",
+            "elementSourceServer": "mds1",
+            "elementSuperTypeNames": [
+                "ComplexSchemaType",
+                "SchemaType",
+                "SchemaElement",
+                "Referenceable",
+                "OpenMetadataRoot"
+            ],
+            "elementTypeDescription": "A schema type for a table oriented data structure.",
+            "elementTypeId": "248975ec-8019-4b8a-9caf-084c8b724233",
+            "elementTypeName": "TabularSchemaType",
+            "elementTypeVersion": 1,
+            "elementVersion": 1,
+            "headerVersion": 0,
+            "status": "ACTIVE"
+        }
+    }
```

Comment:
do we really need all these properties?

Reply:
This json is a superset of all possible fields that can be returned by Egeria during the Get Asset API. This is used mainly to load the relevant classes at compile time. At runtime any subset of these fields can be returned by Egeria and we can  read only the relevant fields. But if we reduce the fields in this file and they are returned by Egeria at runtime, then we can end up in a run time error. That is the purpose of having all the possible fields in this json. It sets up some sort of template response for the REST API response from Egeria.